scene/gui/video_player.cpp

```diff
@@ -186,6 +186,18 @@ void VideoPlayer::_notification(int p_notification) {
 			draw_texture_rect(texture, Rect2(Point2(), s), false);
 
 		} break;
+
+		case NOTIFICATION_PAUSED: {
+			playback->set_paused(true);
+			set_process_internal(false);
+			last_audio_time = 0;
+		} break;
+
+		case NOTIFICATION_UNPAUSED: {
+			playback->set_paused(false);
+			set_process_internal(true);
```

Comment:
Does it mean that the playback is going to be resumed even if a video was paused manually (`$video.paused = true`) by a user? (not via `get_tree().paused`).

Reply:
Xrayez is right, it should differentiate if it was paused manually or not before unpausing it.